src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -122,6 +124,8 @@ private async Task ExecuteAsync(HttpContext context)
             catch (Exception e)
             {
                 await _serializer.WriteErrorAsync(context, "An error occurred while processing the GraphQL query", e);
+                _logger ??= context.RequestServices.GetRequiredService<ILogger<GraphQLMiddleware>>();
+                _logger.LogError("An error occurred while processing the GraphQL query, {e}", e);
```

Comment:
It's better to pass the exception as an exception, because then the logger will decide how to log it, as opposed to doing a `ToString()` right there as a parameter.

```suggestion
                _logger.LogError(e, "An error occurred while processing the GraphQL query.");
```